package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java

```diff
@@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){
 	}
 
 	private Object fromString(String value) {
+		String sysValue = System.getProperty("cloudslang.worker." + name);
+		if(sysValue != null) {
+			value = sysValue;
+		}
```

Comment:
How about this:
String sysValue1 = System.getProperty("cloudslang.worker." + name);
String sysValue2 = System.getProperty("cloudslang.conf." + name);
if(sysValue1 != null) {
   value = sysValue1;
} else if(sysValue2 != null) {
   value = sysValue2;
}


From: Meir Wahnon [mailto:notifications@github.com]
Sent: יום א 05 יולי 2015 12:59
To: CloudSlang/score
Cc: Eskin, Alexander
Subject: Re: [score] Load config value from Java system properties first, then from the xml (#50)


In package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java<https://github.com/CloudSlang/score/pull/50#discussion_r33893478>:

> @@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){

>       }

>

>       private Object fromString(String value) {

> +            String sysValue = System.getProperty("cloudslang.worker." + name);

> +            if(sysValue != null) {

> +                    value = sysValue;

> +            }

You can make this a one liner using:
value = System.getProperty("cloudslang.worker." + name,value);

—
Reply to this email directly or view it on GitHub<https://github.com/CloudSlang/score/pull/50/files#r33893478>.